### PR TITLE
fix:Apple silicon support.

### PR DESCRIPTION
**Pull request by JeffreyMa597**
As the description of the title. This PR is to support packaging for Apple chips.

Ref:https://github.com/Magickbase/neuron-public-issues/issues/62

### CODE REVIEW

**Review by Keith-CY — APPROVED** [review]

File: packages/neuron-wallet/electron-builder.yml [hunk 1]
```diff
@@ -62,8 +62,11 @@ mac:
   entitlementsInherit: assets/entitlements.plist
   electronLanguages: ["en", "zh", "zh_CN", "zh_TW"]
   target:
-    - dmg
-    - zip
+    - target: default
```
Comment (yanguoyu) [inline packages/neuron-wallet/electron-builder.yml:65]: Should we retain `dmg` and `zip`? @Keith-CY

Comment (Keith-CY) [reply packages/neuron-wallet/electron-builder.yml:65]: `target: default` will package a `zip` and a `dmg`

Ref: https://www.electron.build/configuration/mac.html

Comment (JeffreyMa597) [reply packages/neuron-wallet/electron-builder.yml:65]: @yanguoyu It seems this value should be a `String`. 
image

Comment (Keith-CY) [reply packages/neuron-wallet/electron-builder.yml:65]: > @yanguoyu It seems this value should be a `String`. image

Yes, it's a string here

```yml
target:
  - target: default
    arch:
      - arm64
      - x64
```

equals to

```json
{
  target: [{
    target: 'default',
    arch: ['arm64', 'x64']
  }]
}
```

Comment (yanguoyu) [reply packages/neuron-wallet/electron-builder.yml:65]: I have tried with `default` to package, it will output zip and dmg.

**Review by yanguoyu — APPROVED** [review]

**Comment by JeffreyMa597** [comment]
Cause the Neuron didn't accept commits without verifying, I'll create a new branch to submit these changes based on the latest `develop` branch. Thank you.

**Pull request closed without merging**